### PR TITLE
Preview/delete test

### PR DESCRIPTION
**Pull request by sahma19**

### CODE REVIEW

File: domains/transfer/Transfer.API/API.IntegrationTests/Factories/TransferAgreementsApiWebApplicationFactory.cs [hunk 1]
```diff
@@ -38,7 +38,7 @@ public class TransferAgreementsApiWebApplicationFactory : WebApplicationFactory<
     protected override void ConfigureWebHost(IWebHostBuilder builder)
     {
         builder.UseSetting("Otlp:ReceiverEndpoint", OtlpReceiverEndpoint);
-        builder.UseSetting("ConnectionInvitationCleanupService:SleepTime", "00:00:02");
+        builder.UseSetting("ConnectionInvitationCleanupService:SleepTime", "00:00:03");
```
Comment (tnickelsen) [inline domains/transfer/Transfer.API/API.IntegrationTests/Factories/TransferAgreementsApiWebApplicationFactory.cs:41]: Why this change?

File: domains/transfer/Transfer.API/API/Dockerfile [hunk 2]
```diff
@@ -23,5 +23,5 @@ RUN apt-get -y update && apt-get -y install wget && apt-get clean
 WORKDIR /app
 COPY --from=publish /app/publish .
 COPY migrations /migrations
-HEALTHCHECK CMD wget -qO- http://localhost:80/healthz > /dev/null || exit 1
+HEALTHCHECK CMD wget -qO- http://localhost:80/health > /dev/null || exit 1
```
Comment (tnickelsen) [inline domains/transfer/Transfer.API/API/Dockerfile:26]: Did you check that this works?

File: domains/transfer/Transfer.API/API/Options/ConnectionInvitationCleanupServiceOptions.cs [hunk 3]
```diff
@@ -7,6 +7,6 @@ public class ConnectionInvitationCleanupServiceOptions
 {
     public const string Prefix = "ConnectionInvitationCleanupService";
 
-    [Required]
+    [Range(typeof(TimeSpan), "00:00:02", "02:00:00")]
```
Comment (tnickelsen) [inline domains/transfer/Transfer.API/API/Options/ConnectionInvitationCleanupServiceOptions.cs:10]: Why this? The required flag forces you to set it

File: domains/transfer/Transfer.API/API.IntegrationTests/Controllers/ConnectionsControllerTests.cs [hunk 4]
```diff
@@ -66,4 +67,74 @@ public async void GetConnections_ShouldOnlyReturnConnectionsInvolvingTheCompany(
...
+        var myCompanyId = Guid.NewGuid();
+        var client = factory.CreateAuthenticatedClient(sub: myCompanyId.ToString());
+
+        var response = await client.DeleteAsync($"api/connections/{nonExistentConnectionId}");
+
+        response.StatusCode.Should().Be(System.Net.HttpStatusCode.NotFound);
+    }
+
+    [Fact]
+    public async Task DeleteConnection_ShouldReturnNotFound_WhenUserIsUnauthorized()
```
Comment (tnickelsen) [inline domains/transfer/Transfer.API/API.IntegrationTests/Controllers/ConnectionsControllerTests.cs:84]: Shouldnt this be 401 Unauthorized?

**Review by tnickelsen — CHANGES_REQUESTED** [review]

File: domains/transfer/Transfer.API/API/Dockerfile (hunk 2 above)
Comment (sahma19) [reply domains/transfer/Transfer.API/API/Dockerfile:26]: Yes

File: domains/transfer/Transfer.API/API/Options/ConnectionInvitationCleanupServiceOptions.cs (hunk 3 above)
Comment (sahma19) [reply domains/transfer/Transfer.API/API/Options/ConnectionInvitationCleanupServiceOptions.cs:10]: The required flag does nothing, to enforce it during runtime. It is omitted, as the code enforcing the presence of this variable exists in program.cs

**Review by tnickelsen — APPROVED** [review]
